test/functional/conf_args.py

```diff
@@ -37,13 +37,13 @@ def run_test(self):
         os.mkdir(new_data_dir)
         self.start_node(0, ['-conf='+conf_file, '-wallet=w1'])
         self.stop_node(0)
-        assert os.path.isfile(os.path.join(new_data_dir, 'regtest', 'wallets', 'w1'))
+        assert os.path.exists(os.path.join(new_data_dir, 'regtest', 'wallets', 'w1'))
```

Comment:
isdir?

Reply:
Seems worse, makes test more fragile than it needs to be.

Reply:
Yeah, agree, not relevant here.